### PR TITLE
Add hesperos recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Have not heard from the author @chgodard in a week.

The author can be added later on as maintainer.

Moving to ready for review.

File: recipes/hesperos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - napari <0.5.0
```
Comment (jaimergp) [inline recipes/hesperos/meta.yaml:22]: Upstream says `napari <0.4.15` :/

Comment (goanpeca) [reply recipes/hesperos/meta.yaml]: Fixed!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Pull request merged**